db-tools-ij/src/main/grammar/ij.jj

```diff
@@ -860,19 +814,13 @@ class ij {
             rs = dbmd.getProcedureColumns(null,schema,proc,null);
 
             // Small subset of the result set fields available
-            int[] displayColumns = new int[] {
-                rs.findColumn("COLUMN_NAME"),
-                rs.findColumn("TYPE_NAME"),
-                rs.findColumn("ORDINAL_POSITION")
+            int[] columnParameters = new int[] {
+                rs.findColumn("COLUMN_NAME"), 32, 0,
+                rs.findColumn("TYPE_NAME"), 16, 32,
+                rs.findColumn("ORDINAL_POSITION"), 16
```

Comment:
Is there one more number missing after 16? I didn't try it but it feels like index arithmetic like `i*3+2` later would be messed up.

Reply:
good catch!

Reply:
i thought i can do this in a "simple" way but that was too simple. i fixed this now with a builder-pattern